app/controllers/AdminUsers/orders_controller.rb

```diff
@@ -0,0 +1,13 @@
+module AdminUsers
+  class OrdersController < BaseController
+    def index
+      render :index, locals: { orders: Order.where(employee: employee), employee: employee }
```

Comment:
why not `Order.includes`?

Reply:
I implemented this kind of code in order to provide for admin separate lists of rewards bought by each employee separately. I understood in this way the ac: 'Admin can see a list of all the rewards bought by **an employee**'. Therefore I think that I don't need `.includes` because the method choose only one employee already at start. Of course I tried to add `.includes` in my code, but after that bullet gem yelled that it should be removed.

Reply:
I implemented this kind of code in order to provide for admin separate lists of rewards bought by each employee separately. I understood in this way the ac: 'Admin can see a list of all the rewards bought by an employee'. Therefore I think that I don't need .includes because the method choose only one employee already at start in order to show his email only on top of the page, not in details of each bought rewards. Of course I tried to add .includes in my code, but after that bullet gem yelled that it should be removed.

Reply:
`includes` is indeed unnecessary in this case, but  **User story** says: _As an admin, I can navigate to the employee's list with the history of rewards..._ so in my understanding those rewards should be listed on the employee's index page